docs/hosting/securing/hardening-task-runners.md

```diff
@@ -0,0 +1,14 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Hardening task runners
+description: "Harden task runners for better isolation for your self-hosted n8n instance."
+contentType: howto
+---
+
+# Hardening task runners
+
+[Task runners](/hosting/configuration/task-runners/) are responsible for executing code from the [Code node](/integrations/builtin/core-nodes/n8n-nodes-base.code/). While Code node executions are secure, you can follow these recommendations to further harden your task runners.
+
+## Run task runners as sidecars in external mode
+
+To increase the isolation between the core n8n process and code in the Code node, run task runners in [external mode](/hosting/configuration/task-runners#setting-up-external-mode). External task runners launch as separate containers, providing a sandboxed environment to execute the JavaScript defined in the Code node.
```

Comment:
Both modes provide a sandboxed execution environment. Could we rephrase this to something like

```suggestion
To increase the isolation between the core n8n process and code in the Code node, run task runners in [external mode](/hosting/configuration/task-runners#setting-up-external-mode). External task runners launch as separate containers, providing a fully isolated environment to execute the JavaScript defined in the Code node.
```
?

Meaning `internal` mode is partially isolated (i.e. they share the same host/container and filesystem), and `external` is fully isolated (separate hosts/containers, no filesystem access)

Reply:
Yep, that works for me! Thanks for clarifying.